packages/xo-server/src/_createNdJsonStream.js

```diff
@@ -0,0 +1,22 @@
+import asyncIteratorToStream from 'async-iterator-to-stream'
+
+function* values(object) {
+  const keys = Object.keys(object)
+  for (let i = 0, n = keys.length; i < n; ++i) {
```

Comment:
Why not using  `for .. in`?

Reply:
Because `for .. in` iterates over inherited properties, which is not not what I want.